MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj

```diff
@@ -37,10 +37,9 @@
   <PropertyGroup>
     <ProjectGuid>28b258ca-5ecc-41ee-a5b0-c2f24f3a4984</ProjectGuid>
     <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>

```

Comment:
>18362 [](start = 35, length = 5)

Curious Scott, why is this needed or good to do?

Reply:
I changed it to match winui_class_lib, to eliminate the SupportedOSPlatform warning.  winui_class_lib does not build with 17763, so I had to leave it at 18362, but for consistency all minvers should probably also be 18362.